tests/test_validators.py

```diff
@@ -48,3 +54,301 @@ def test_value_valid(self):
     def test_value_invalid_type(self):
         with self.assertRaises(exceptions.FieldError):
             self.validate(fake.word())
+
+
+class ValidatePositiveNumberValueTestCase(unittest.TestCase):
+    def setUp(self):
+        self.positive_int_value = fake.random_int(min=1)
+        self.not_positive_int_value = fake.random_int(min=1) * (-1)
+        self.positive_float_value = fake.pyfloat(positive=True)
+        self.not_positive_float_value = fake.pyfloat(positive=True) * (-1)
+
+    def validate(self, value, value_type):  # pylint:disable=no-self-use
```

Comment:
1. pylint:disable is unjustified here. (Could use `@classmethod`)
1. Only effect of this method is to rename `validators.is_number_positive` -> `self.validate`. IMO it should be removed

Reply:
Removed because of validators changes.